opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -262,9 +262,21 @@ namespace Opm
             return;
         }
 
-        BVectorWell xw(1);
-        this->linSys_.recoverSolutionWell(x, xw);
-        updateWellState(simulator, xw, well_state, deferred_logger);
+
+        try{
+            BVectorWell xw(1);
+            this->linSys_.recoverSolutionWell(x, xw);
+
```

Comment:
Nitpick: Extra blank. (Also space before brace at `try` and parenthesis at `catch`.)